litmus-portal/backend/graphql-server/pkg/project/create.go

```diff
@@ -0,0 +1,36 @@
+package project
```

Comment:
We are having all the mutations, queries and subscription helper functions in the graphql package.

Reply:
Yes but having them all under one roof will not be advisable as it can complicate the code readability